test/containers/HoFPageSpec.js

```diff
@@ -1,39 +1,30 @@
-import setup from '../setup.js'
-setup()
-
 import test from 'tape'
 import React from 'react'
 import { MemoryRouter } from 'react-router-dom'
-import { createStore, applyMiddleware, compose } from 'redux'
 import { Provider } from 'react-redux'
-import thunk from 'redux-thunk'
+import { mount } from 'enzyme'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'enzyme'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->